diff --git a/code/qcommon/q_shared.h b/code/qcommon/q_shared.h
index e4c98f6..2810e87 100644
--- a/code/qcommon/q_shared.h
+++ b/code/qcommon/q_shared.h
@@ -34,8 +34,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
   #define HOMEPATH_NAME_UNIX		".openarena"
   #define HOMEPATH_NAME_WIN		"OpenArena"
   #define HOMEPATH_NAME_MACOSX		HOMEPATH_NAME_WIN
-  #define GAMENAME_FOR_MASTER		"OpenArenaAndroid"	// must NOT contain whitespace.  No servers show up if you use "openarena"
-  //#define LEGACY_PROTOCOL		1 // OA still uses the legacy protocol // OA for Android have no use of the old servers anyway
+  #define GAMENAME_FOR_MASTER		"Quake3Arena"	// must NOT contain whitespace.  No servers show up if you use "openarena"
+  #define LEGACY_PROTOCOL		1 // OA still uses the legacy protocol
 #else
   #define PRODUCT_NAME			"ioq3"
   #define BASEGAME			"baseq3"
